test/cypress/integration/consumer-tools/before-you-claim/before-you-claim.cy.js

```diff
@@ -27,4 +31,957 @@ describe('Planning your Social Security', () => {
     cy.wait('@getSpanish');
     cy.url().should('contain', '/es');
   });
+
+  it('should display tool tip in step one after clicking svg tip icon', () => {
+    cy.get('#claiming-social-security #tooltip-container').should(
+      'not.be.visible'
+    );
+    cy.get(
+      '#claiming-social-security #step-one-form .cf-icon-svg__help-round'
+    ).click();
+    cy.get('#claiming-social-security #tooltip-container').should('be.visible');
+  });
+
+  it('should show error if user is over the age of 70', () => {
+    claim.enterAgeOver70();
+    cy.get(
+      '#claiming-social-security #step-one-form .m-notification__warning'
+    ).should('be.visible');
+  });
+
+  it('should display estimated benefits', () => {
+    claim.enterAgeUnder50();
+    claim.claimGraph().should('be.visible');
+    cy.get('#claiming-social-security #graph-container .learn-how a').should(
+      'have.attr',
+      'href',
+      '/consumer-tools/retirement/before-you-claim/about/'
+    );
+  });
+
+  //*********************************************************/
+  //*** Check to see if each bar is showing on the graph ****/
+  it('should display all bars in the graph', () => {
+    claim.enterAgeUnder50();
+    cy.get('#claiming-social-security #claim-canvas .graph__bar').each(
+      (elem) => {
+        cy.wrap(elem).should('not.have.css', 'height', '0px');
+      }
+    );
+  });
+
+  it('should only display bars >= age 65 in the graph', () => {
+    claim.enterAgeOver50();
+    cy.get('#claiming-social-security #claim-canvas .graph__bar').each(
+      (elem, index) => {
+        switch (index) {
+          case 0:
+          case 1:
+          case 2:
+            cy.wrap(elem).should('have.css', 'height', '0px');
+            break;
+          case 3:
+          case 4:
+          case 5:
+          case 6:
+          case 7:
+          case 8:
+            cy.wrap(elem).should('not.have.css', 'height', '0px');
+            break;
+        }
+      }
+    );
+  });
+
+  //*************************************************/
+  // Testing answers to "Are you married" question in step 2
+  it('show correct lifestyle response to yes button', () => {
+    claim.enterAgeUnder50();
+    cy.get('#claiming-social-security .question-married .lifestyle-btn.a-btn')
+      .contains('Yes')
+      .click();
+    cy.get('#claiming-social-security .question-married .lifestyle-response')
+      .should('have.attr', 'data-responds-to', 'yes')
+      .and('be.visible')
+      .next()
+      .should('have.attr', 'data-responds-to', 'no')
+      .and('not.be.visible')
+      .next()
+      .should('have.attr', 'data-responds-to', 'widowed')
+      .and('not.be.visible');
+  });
```

Comment:
Should this formatted the same as `it('show correct lifestyle response to no button', () => {`, utilizing `lsQuestions`, or is it fine like this?

Reply:
Agreed, it should be uniform with the other code. This was from the initial attempt, which worked for the first paragraph but not the others.